.github/workflows/continuous-integration.yml

```diff
@@ -66,20 +73,21 @@ jobs:
         with:
           node-version: "${{ steps.nvm.outputs.NVMRC }}"
 
-      - name: Setup cache
+      - name: Setup npm cache
         uses: pat-s/always-upload-cache@v1.1.4
         with:
           path: node_modules
-          key: ${{ runner.OS }}-npm-${{ hashFiles('**/package-lock.json') }}
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
-            ${{ runner.OS }}-npm-${{ env.cache-name }}-
-            ${{ runner.OS }}-npm-
-            ${{ runner.OS }}-
+            ${{ runner.os }}-npm-${{ env.cache-name }}-
+            ${{ runner.os }}-npm-
+            ${{ runner.os }}-
```

Comment:
Curious: why are the last restore keys under each cache necessary: `${{ runner.os }}-`
Wouldn't just `${{ runner.os }}-npm-` suffice?

Reply:
[The docs](https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows) explain it a bit more in detail.

In theory we could omit `${{ runner.os }}-` as we don't use multiple environments, so just `npm-` would work. But they use that in their examples as well, as some sort of last resort I suppose.